ui/settings.glade

```diff
@@ -4243,8 +4243,9 @@ If available select &lt;small&gt;&lt;tt&gt;pulse&lt;/tt&gt;&lt;/small&gt; as inp
                                             <property name="can_focus">False</property>
                                             <property name="active">0</property>
                                             <items>
+                                              <item id="16000">16000 Hz</item>
                                               <item id="44100">44100 Hz</item>
-                                              <item id="91000">91000 Hz</item>
+                                              <item id="96100">96000 Hz</item>
```

Comment:
Sorry to request one more change, but I noticed that the item id is set inconsequently for the 96000 Hz sample rate.
```suggestion
                                              <item id="96000">96000 Hz</item>
```